spec/spec_helper.rb

```diff
@@ -51,6 +51,14 @@ class ActionController::TestCase
     attribute :revenue, :integer
   end
 
+  strong_resource :parent_company do
+    attribute :title, :string
+  end
+
+  strong_resource :unicorn do
```

Comment:
updated specs to use unicorn. i want to test declaring strong_resource without block.